### DISCUSSION TITLE
UxPlay 1.67

### DISCUSSION THREAD

**Discussion by fduncanh** [FDH2/UxPlay#247] [Announcements]
UxPlay is a GPLv3 unix AirPlay2 Mirror and AirPlay2 Audio (but not AirPlay2 Video) server: its main use is to act like an AppleTV for screen-mirroring (with audio) of iOS/MacOS clients (iPads, iPhones, MacBooks, etc.) in a shareable window on the server display,  on a host running Linux, macOS, or other unix, or on Windows 10/11 (with the unix-like MYSYS2 environment), using Apple's AirPlay2 Mirror protocol. When the client screen is not mirrored, Apple Lossless (ALAC) AirPlay2 Audio can be streamed from the client to the server (with cover art made available for display by an external file viewer).

UxPlay is derived from https://github.com/FD-/RPiPlay, with audio and video rendered using GStreamer, and many enhancements. While originally targeted at x86/x86_64 server and desktop systems, it also supports Raspberry Pi  (with accelerated hardware video decoding using Video4Linux2 on models 4B and earlier).  UxPlay should run on any Linux, and is tested on many Linux distributions.  Also tested on macOS Sonoma and earlier (Intel and M2), FreeBSD 14, Windows 10 and 11 (64 bit). On Raspberry Pi, it is tested on Raspberry Pi OS (Bullseye and Bookworm, 32- and 64-bit), Ubuntu, Manjaro RPi4  and  openSUSE.

ChangeLog:
Added support for Apple-style one-time pin authentication of clients (new option "-pin").  Removed rpi* options (which are not valid with new Raspberry Pi model 5,  and can be replaced by combinations of other options).  Updated llhttp to v9.1.3;  various other minor fixes and improvements (user-specifiable MAC address, export  helper data for remotes, detection of unsupported h265/HEVC video).

<hr /><em>This discussion was created from the release <a href='https://github.com/FDH2/UxPlay/releases/tag/v1.67'>UxPlay 1.67</a>.</em>

**Comment by shuax** [comment]
Great work
